.github/workflows/lint.yml

```
@@ -24,7 +24,16 @@ jobs:
             .ts
             .js
 
+      - name: install node v12
```

Let's make this fancier
```suggestion
      - name: Setup Node 👷
```